### ISSUE TITLE
schema.org pages for types in pending.schema.org lack HTML title information, use boilerplate description

### ISSUE THREAD

**Issue by Aaranged** [schemaorg/schemaorg#1792]
Don't know whether or not this is somehow deliberate or a bug, but broad best practices for the web are to have unique `<title>` and `<meta> description`  values for unique resources.

E.g.: http://schema.org/CriticReview

```html
<title>- schema.org</title>
<meta name="description" content="Schema.org is a set of extensible schemas that enables webmasters to embed structured data on their web pages for use by search engines and other applications." />
```

Compare with: http://pending.schema.org/CriticReview

```html
<title>CriticReview - pending.schema.org</title>
<meta name="description" content="Schema.org Type: CriticReview - A CriticReview is a more specialized form of Review written or published by a source that is recognized for its reviewing activities. These can include online columns, travel and food guides, TV and radio shows, blogs and other independent Web sites..." />
```

**Comment by RichardWallis** [comment]
Issue is that <title> is not correctly populated on the page presented when clicking on a term that is in an extension, when not in that extension.

**Comment by RichardWallis** [comment]
Fixed awaiting release

**Comment by RichardWallis** [comment]
Released in V3.4
